pkg/vmimage/data/script.sh

```diff
@@ -207,10 +207,15 @@ sed -i -e 's/^ResourceDisk.Format=.*/ResourceDisk.Format=n/' /etc/waagent.conf
 rpm -q kernel --last | sed -n '1 {s/^[^-]*-//; s/ .*$//; p}' >/var/tmp/kernel-version
 rpm -q atomic-openshift-node --qf '%{VERSION}-%{RELEASE}.%{ARCH}' >/var/tmp/openshift-version
 
+# enabling kdump
+grub2-mkconfig -o /boot/grub2/grub.cfg
+systemctl enable kdump.service
```

Comment:
this just has the effect of creating a symlink -- please check that symlink into pkg/vmimage/data/etc/systemd/system/multi-user.target.wants instead